master/buildbot/www/oauth2.py

```diff
@@ -188,6 +188,35 @@ def getUserInfoFromOAuthClient(self, c):
                     email=data["email"],
                     avatar_url=data["picture"])
 
+class GitHubEntAuth(OAuth2Auth):
+    def __init__(self,
```

Comment:
can you not extend GitHubAuth instead ? the github reporter uses the same code for both : https://github.com/buildbot/buildbot/blob/58ac4c7adc1896842c70f9dee1c85a479da95a08/master/buildbot/reporters/github.py#L47